### PR TITLE
Add PDO MyQL SSL support

### PR DESCRIPTION
**Pull request by gfleury**
Add PDO MyQL SSL support

### CODE REVIEW

**Comment by fredemmott** [comment]
Closing for now (https://github.com/facebook/hhvm/wiki/Human-Timeouts) - will re-open when comments on reviews.facebook.net are addressed.

**Comment by Orvid** [comment]
@gfleury Any plans to fix this? If not, does anyone know the specific tests that this was failing?

**Comment by Kazanir** [comment]
@Orvid If you check the Phabricator comments it looks like @paulbiss just wanted some unit tests *written*, not that it was failing existing ones?

**Comment by slaughter550** [comment]
@fredemmott does this just require unit tests or what's holding up this PR? I'd be happy to work on it if someone tells me whats needed

**Comment by aorenste** [comment]
I'll take a look to see what the problems were

**Comment by slaughter550** [comment]
@aorenste thank you!

**Comment by slaughter550** [comment]
@gfleury Thanks a lot! Does anyone know when this will be released?

**Comment by aorenste** [comment]
@slaughter550 This should be available in 3.17 (or the nightly builds).

File: hphp/runtime/ext/pdo_mysql.cpp [hunk 1]
```diff
@@ -389,6 +390,27 @@ bool PDOMySqlConnection::create(const Array& options) {
...
+                              nullptr);
+    ssl_cert = pdo_attr_strval(options, PDO_MYSQL_ATTR_SSL_CERT,
+                               nullptr);
+    ssl_cipher = pdo_attr_strval(options, PDO_MYSQL_ATTR_SSL_CIPHER,
+                                 nullptr);
+
+
+    if ((ssl_ca || ssl_capath || ssl_key || ssl_cert || ssl_cipher) && 
+         !host.same(s_localhost)) {
+      if (mysql_ssl_set(m_server, ssl_key.c_str(), ssl_cert.c_str(),
```
Comment (aorenste) [inline hphp/runtime/ext/pdo_mysql.cpp]: I think your concern is addressed in PR #7423

**Pull request closed without merging**